mycore-sass/src/main/java/org/mycore/sass/MCRResourceImporter.java

```diff
@@ -0,0 +1,140 @@
+            return null;
+        }
+        String resourcePath = url.substring(SASS_URL_PREFIX.length());
+        URL resource = getWebResourceUrl(resourcePath);
+        if (resource == null || resource.getFile().endsWith("/") || !URL_HELPER.isFile(resource)) {
+            //with jetty the directory does not end with /, so we need to REAL check
+            return null;
+        }
+        LOGGER.debug("Resolved {} to {}", url, resource);
+        return resource == null ? null : url;
+    }
+
+    /**
+     * Handles the import of a URL and returns the success of the import along with its content.
+     *
+     * This method takes a URL as input and checks if it starts with the "sass:/" prefix. If it does,
+     * it attempts to locate the resource URL. If the resource URL is found,
+     * it reads its content returns the an ImportSuccess object.
+     *
+     * @param url The URL of the resource to import.
+     * @return An ImportSuccess object containing the content and syntax of the imported resource.
+     * @throws IOException If there is an error in handling the import.
+     */
+    @Override
+    public ImportSuccess handleImport(String url) throws IOException {
+        LOGGER.info("handle import: " + url);
+        if (!url.startsWith(SASS_URL_PREFIX)) {
+            return null;
+        }
+        ImportSuccess.Builder result = ImportSuccess.newBuilder();
+        String resourcePath = url.substring(SASS_URL_PREFIX.length());
+        LOGGER.info("resource: " + resourcePath);
+        URL configResource = getWebResourceUrl(resourcePath);
+        LOGGER.info("resource url: " + configResource);
+        URLConnection urlConnection = configResource.openConnection();
+        try (InputStream in = urlConnection.getInputStream()) {
+            ByteString content = ByteString.readFrom(in);
+            result.setContentsBytes(content);
+            result.setSyntax(SyntaxUtil.guessSyntax(urlConnection));
+        }
+        return result.build();
+    }
+
+    /**
+     * Retrieves the URL of a web resource given its path.
+     *
+     * @param resourcePath The path of the web resource to retrieve.
+     * @return The URL of the web resource.
+     */
+    private URL getWebResourceUrl(String resourcePath) {
+        return MCRDeveloperTools.getOverriddenFilePath(resourcePath, true)
+            .map(p -> {
+                try {
+                    return p.toUri().toURL();
+                } catch (MalformedURLException e) {
+                    throw new UncheckedIOException(e);
+                }
+            })
+            .or(() -> Optional.ofNullable(MCRConfigurationDir.getConfigResource("META-INF/resources/" + resourcePath)))
+            .orElseGet(() -> {
+                try {
+                    return servletContext.getResource("/" + resourcePath);
+                } catch (MalformedURLException e) {
+                    throw new UncheckedIOException(e);
+                }
+            });
+    }
```

Comment:
needs to be replaced by a central resource locating method. @toKrause anything ready yet?

Reply:
Draft PR: https://github.com/MyCoRe-Org/mycore/pull/1849

Reply:
This would be replaced by `MCRResourceHelper#getWebResourceUrl(resourcePath)`. I will do this once thes PR is merged and I rebase my PR.